libraries/AP_HAL_ChibiOS/HAL_ChibiOS_Class.cpp

```diff
@@ -254,6 +255,12 @@ static void main_loop()
         }
 #endif
         schedulerInstance.watchdog_pat();
+#if !defined(HAL_BOOTLOADER_BUILD) && !defined(IOMCU_FW)
+        static uint32_t stm32_old_fpe_flags;
```

Comment:
don't need this variable now, can just check against zero

Reply:
Fixed.